tests/unit/helpers.js

```diff
@@ -224,12 +224,12 @@ export const fakeReview = Object.freeze({
   body: 'It is Okay',
 });
 
-export const fakeShelf = Object.freeze({
+export const fakeExternalShelf = Object.freeze({
   title: createLocalizedString('Top Rated Themes'),
   url:
     'https://addons-dev.allizom.org/api/v5/addons/search/?sort=rating&type=statictheme',
   endpoint: 'search-themes',
-  addonType: ADDON_TYPE_STATIC_THEME,
+  addon_ype: ADDON_TYPE_STATIC_THEME,
```

Comment:
😬

Reply:
Also, this is still passing? That's concerning.

Reply:
SERIOUSLY? 🤦

Reply:
I think it's passing because we override `addon_type` every time we use this fake object.